css/fanbool.css

```diff
@@ -317,7 +377,7 @@ print {
     }
 
     nav li a {
-        padding: 0.1em 0.75em;
+        padding: 0.1em 0.5em;
```

Comment:
With the extra menu items, we need a little more compression before jumping to mobile.